build/repo.targets

```diff
@@ -119,7 +119,7 @@
   </Target>
 
   <Target Name="RunNativeTest" DependsOnTargets="GetToolsets">
-    <Exec Command="&quot;$(VisualStudioMSBuildx86Path)&quot; &quot;$(RepositoryRoot)test\CommonLibTests\CommonLibTests.vcxproj&quot; /t:Test $(BuildArgs) -p:Platform=%(Platforms.VCPlatform)"
+    <Exec Command="&quot;$(VisualStudioMSBuildx86Path)&quot; &quot;$(RepositoryRoot)test\CommonLibTests\CommonLibTests.vcxproj&quot; /t:Test $(BuildArgs) -p:Platform=x64"
```

Comment:
Why?

Reply:
1. Only wanted to run tests once.
2. I'll need to fix some linking issue with obj files.